File: spring-best/src/main/java/com/best/springbest/model/Color.java
package com.best.springbest.model;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "colors")
public class Color {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String name;

}

File: BEST-App-main/lib/model/color.dart
class Colorss {
  final int id;
  final String name;

  Colorss(this.id, this.name);

  static Map<String, dynamic> toJson(Colorss color) {
    return {
      'id': color.id,
      'name': color.name,
    };
  }
}
